src/components/task-plan/homework/exercises.cjsx

```diff
@@ -33,6 +33,9 @@ ExerciseCardMixin =
   renderExercise: ->
     content = @props.exercise.content
     question = content.questions[0]
+    question.answers = _(question.answers).chain()
+      .sortBy('id')
+      .value()
     renderedAnswers = _.map(question.answers, @renderAnswers)
```

Comment:
Yes that's a good way to combine;  though map must be called w/o the list param:
```
renderedAnswers = _(question.answers).chain()
.sortBy('id')
.map(@renderAnswers)
.value()
```